tls/s2n_client_finished.c

```diff
@@ -25,39 +25,30 @@
 
 #include "utils/s2n_safety.h"
 
+S2N_RESULT s2n_finished_recv(struct s2n_connection *conn, uint8_t *our_version);
```

Comment:
Not super fond of the convention of "our_version" and "their_version". We don't have any better names for those?